cloud/src/autogluon/cloud/predictor/cloud_predictor.py

```diff
@@ -54,7 +68,6 @@ class CloudPredictor(ABC):
     def __init__(
         self,
         cloud_output_path,
```

Comment:
We likely won't because of political issue. When it comes to we are going to support a different backend. We could probably turn the current CloudPredictor, which is not user facing as user will use `TabularCloudPredictor` for example, into SageMakerCloudPredictor. And create a new interface CloudPredictor that has backend parameter inside the init function. It will just wrap SageMakerCloudPredictor